### PR TITLE
Updated deps + SC is now a peerDependency

### PR DESCRIPTION
**Pull request by chrisvanmook**
## Amsterdam styled components pull request

Before opening a pull request, please ensure:

- [ ] The default export from a file matches the file name
- [ ] You have been following the guidelines, written in the [README](../blob/master/docs/CONTRIBUTING.md#user-content-conventions-and-rules) file
- [ ] Your code has the necessary tests written
- [ ] You have updated the [CHANGELOG.md unreleased sections](../blob/master/CHANGELOG.md#muser-content-unreleased)

Be kind to code reviewers, please try to keep pull requests as small and focused as possible :)

Covers issue #239

### CODE REVIEW

File: packages/asc-core/package.json [hunk 1]
```diff
@@ -26,8 +26,8 @@
     "test": "echo \"Error: run tests from root\" && exit 1"
   },
   "sideEffects": false,
-  "dependencies": {
-    "styled-components": "^4.3.2"
+  "peerDependencies": {
+    "styled-components": ">= 4.4.1"
```
Comment (jonkoops) [inline packages/asc-core/package.json]: This version range also includes `styled-component` v5, do we want that?

Comment (chrisvanmook) [reply packages/asc-core/package.json]: Yes, as v5 doesn't have breaking changes: https://github.com/styled-components/styled-components/releases

Comment (jonkoops) [reply packages/asc-core/package.json]: :ok_hand: :ok:

**Comment by jonkoops** [comment]
@chrisvanmook You can also `git pull --rebase origin master`, that removes the merge commits.

**Pull request merged**